geode-connectors/src/acceptanceTest/java/org/apache/geode/connectors/jdbc/rules/PostgresConnectionRule.java

```diff
@@ -12,7 +12,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.test.junit.rules;
+package org.apache.geode.connectors.jdbc.rules;
```

Comment:
I am ok with the package name for rules. 

Just want to remind that cq defines its rules in package: ./geode-cq/src/test/java/org/apache/geode/test/dunit/rules

Reply:
Are you suggesting `org.apache.geode.connectors.jdbc.test.junit.rules`?

Reply:
kind of. If it would not cause trouble.